jsonrpc/types.go

```diff
@@ -210,9 +210,9 @@ func (a argBig) MarshalText() ([]byte, error) {
 	return []byte("0x" + b.Text(16)), nil
 }
 
-func argAddrPtr(a types.Address) *types.Address {
-	return &a
-}
+//func argAddrPtr(a types.Address) *types.Address {
```

Comment:
Please remove this comment block if it's not needed

Reply:
Fixed in commit cbc116c8211dc3ecd4685616b182a8d6db4cb005